media/css/firefox/mobile.scss

```diff
@@ -872,7 +872,7 @@ html[dir="rtl"] {
 }
 
 // very basic styling for no-JS visitors
-#mobile-download-buttons-wrapper {
+.mobile-download-buttons-wrapper {
     background: #111;
```

Comment:
Also, I think we still need to keep the app store badges as the non-JS default. So the baseline is to show the app store badges and hide the modal buttons, and when JS is active we can show the modal buttons and hide the store badges.

Then in addition to that, when JS is available and we detect the platform, we can hide the modal button and show just the app store badge for the given platform (ios or andoid).

This is mostly just a matter of getting the cascade right with the different classes that JS appends.

Reply:
Oh another thought... if we're changing the non-JS state the modal buttons shouldn't be links since they won't link to anything, and they'll be totally JS-dependent. We should switch them to `button type="button"` elements.

Reply:
@craigcook thanks for this awesome feedback! Added a commit addressing these.